gulpfile.js

```diff
@@ -36,7 +36,7 @@ const options = {
         query: ['src/standalone.js', 'src/scripts/apploader.js']
     },
     css: {
-        query: ['src/**/*.css', 'src/**/*.scss']
+        query: ['src/**/*.css', 'src/**/*.scss', '!src/stylesheets/**/*']
```

Comment:
Ignores the folder from outputting empty css files

Should have open an issue to decide the folder name, but since the change is small, it could be easily updated